src/core/static/_locales/en_US/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "extensionName": {
-    "message": "Bergamot Translate",
+    "message": "Firefox Translations",
```

Comment:
Same comment as above, but also this highlights the need for different localization strings for different builds